assets/js/state/sagas/sapSystems.test.js

```diff
@@ -1,10 +1,21 @@
 import { recordSaga } from '@lib/test-utils';
 import {
+  applicationInstanceDeregistered,
   sapSystemDeregistered,
   sapSystemUpdated,
 } from '@state/sagas/sapSystems';
-import { removeSAPSystem, updateSAPSystem } from '@state/sapSystems';
-import { sapSystemFactory } from '@lib/test-utils/factories';
+import {
+  removeSAPSystem,
+  removeApplicationInstance,
+  updateSAPSystem,
+  removeDatabaseInstanceFromSapSystem,
+} from '@state/sapSystems';
+import { databaseInstanceDeregistered } from '@state/sagas/databases';
```

Comment:
Why are you importing and testing this here?
This should go in the `sagas/database.test.js` , or not?

Reply:
yes, you are totally right, it's the database saga which dispatches this and hence should be tested there. Same for the other wrongly placed test.